### PR TITLE
HARVESTER: add backup healthy check

### PR DESCRIPTION
**Pull request by WuJun2016**
### Summary
Fixes # 
- https://github.com/harvester/harvester/issues/2631

### Areas or cases that should be tested

- https://github.com/harvester/tests/issues/466

### CODE REVIEW

File: pkg/harvester/l10n/en-us.yaml [hunk 1]
```diff
@@ -545,6 +545,11 @@ harvester:
         prefix: Click
         middle: here
         suffix: to view the backup config.
+      testConnect:
+        actionLabel: Test connection
+        waitingLabel: Test connection...
+        successLabel: Test connection
```
Comment (mantis-toboggan-md) [inline pkg/harvester/l10n/en-us.yaml:551]:
```
waitingLabel: Testing connection...
successLabel: Connection tested
```

**Review by mantis-toboggan-md — CHANGES_REQUESTED** [review]
PR seems to work well, I just have one minor request on the translation strings.

File: pkg/harvester/l10n/en-us.yaml (hunk 1 above)
Comment (WuJun2016) [reply pkg/harvester/l10n/en-us.yaml]: Done

**Comment by WuJun2016** [comment]
@mantis-toboggan-md  To give the QA more time to test, I want to merge the PR first.  thanks

**Pull request merged**